azure-container-apps/ReadMe.md

```diff
@@ -534,23 +498,3 @@ Logs for a container app can be viewed from the **Log Stream**, there is a separ
 7. Test logging into your SCIM bridge URL with the new `bearer token`.
 8. Update your identity provider configuration with the new bearer token.
 </details>
-
-## Update 1Password SCIM Bridge
```

Comment:
Where did this get moved to?

Reply:
[Line 250 - 272](https://github.com/1Password/scim-examples/blob/562c4fdcc8d8e7a5e2406a715d0b83b6cd264bb2/azure-container-apps/ReadMe.md?plain=1#L250-L272)
I was second guessing myself, as it wasn't seeing it to being with either.